docs/nightly/en/reference/sql/data-types.md

```diff
@@ -138,13 +138,19 @@ SELECT now() - INTERVAL '5 minute';
 
 Note that you can also input the interval type using the `INTERVAL 'literal'` format. Using the syntax `'-1 hour -5 minute'::INTERVAL` explicitly casts the string to an interval type, which is how SQL handles type conversion.
 
-GreptimeDB also supports shorthand forms without spaces, such as `3y2mon4h`, but they must be written in the `INTERVAL 'literal'` format:
+GreptimeDB also supports shorthand forms without spaces, such as `3y2mon4h`:
```

Comment:
please also update here to zh, thank you very much

Reply:
Updated.